backend/src/main/java/codesquad/airbnb/accomodation/domain/ProvideName.java

```diff
@@ -0,0 +1,33 @@
+package codesquad.airbnb.accomodation.domain;
+
+public enum ProvideName {
+    CAPACITY("최대 인원"),
+    BED_ROOM("침실"),
+    BED("침대"),
+    WASH_ROOM("욕실"),
+    KITCHEN("주방"),
+    WIFI("무선 인터넷"),
+    AIR_CONDITIONER("에어컨"),
+    HAIR_DRYER("헤어드라이어");
+
+    private final String name;
```

Comment:
👍

Reply:
'name'은 enum 객체의 인스턴스 변수 name이 'name'이니
enum 클래스 명을 변경하는건 어떨까요

```suggestion
public enum ProvideType {
    CAPACITY("최대 인원"),
    BED_ROOM("침실"),
    BED("침대"),
    WASH_ROOM("욕실"),
    KITCHEN("주방"),
    WIFI("무선 인터넷"),
    AIR_CONDITIONER("에어컨"),
    HAIR_DRYER("헤어드라이어");

    private final String name;
```